The file will have its original line endings in your working directory
diff --git a/configure.ac b/configure.ac
index c983e81..69a6417 100644
--- a/configure.ac
+++ b/configure.ac
@@ -227,9 +227,7 @@ AC_DEFINE_UNQUOTED(VIPS_ICC_DIR,"$profile_dir",[default directory for ICC profil
 # we want largefile support, if possible
 AC_SYS_LARGEFILE
 
-# we use libtool
-LT_INIT
 
 # Checks for programs.
 AC_PROG_AWK
 AC_PROG_CC
@@ -320,19 +317,7 @@ AC_HEADER_STDC
 AC_CHECK_HEADERS([errno.h math.h fcntl.h limits.h stdlib.h string.h sys/file.h sys/ioctl.h sys/param.h sys/time.h sys/mman.h sys/types.h sys/stat.h unistd.h io.h direct.h windows.h])
 
 # uncomment to change which libs we build
-# AC_DISABLE_SHARED
-# AC_DISABLE_STATIC
-AC_LIBTOOL_WIN32_DLL
-AC_CHECK_TOOL(DLLWRAP, dllwrap)
-AC_CHECK_TOOL(DLLTOOL, dlltool)
-AC_CHECK_TOOL(OBJDUMP, objdump)
-AC_CHECK_TOOL(RANLIB, ranlib)
-AC_CHECK_TOOL(STRIP, strip)
-AC_CHECK_TOOL(AR, ar)
-AC_CHECK_TOOL(AS, as)
-AC_CHECK_TOOL(LD, ld)
-AC_PROVIDE([AC_LIBTOOL_WIN32_DLL])
-AC_PROG_LIBTOOL
+AC_DISABLE_SHARED
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_C_RESTRICT
@ -1345,19 +1329,6 @@ AC_CONFIG_FILES([
        libvips/mosaicing/Makefile
        libvips/create/Makefile
        libvips/resample/Makefile
-       cplusplus/include/Makefile
-       cplusplus/include/vips/Makefile
-       cplusplus/Makefile
-       tools/Makefile
-       tools/batch_crop
-       tools/batch_image_convert
-       tools/batch_rubber_sheet
-       tools/light_correct
-       tools/shrink_width
-       test/Makefile
-       test/variables.sh
-       test/test-suite/Makefile
-       test/test-suite/helpers/Makefile
        man/Makefile
        doc/Makefile
        doc/libvips-docs.xml